client/static/css/CodeChatEditor.css

```diff
@@ -207,3 +207,13 @@ body {
     margin-bottom: 0px;
     padding-bottom: 0px;
 }
+
+.CodeChat-TinyMCE-TableDefault { 
```

Comment:
Add a comment with a hyperlink back to the `visual_table_class` in `tinymce-webpack.mts` to help explain where it's used.

Reply:
added in [06e78c7](https://github.com/bjones1/CodeChat_Editor/pull/24/commits/06e78c728b3b0cbeab4203e7ee907525d4d26e78)